test/test_view/test_enrichment.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from default import db, with_context
+from factories import ProjectFactory
+from helper import web
+from pybossa.repositories import ProjectRepository
+
+project_repo = ProjectRepository(db)
+
+
+class TestEnrichment(web.Helper):
+
+    @with_context
+    def test_post_enirchment_config(self):
+        project = ProjectFactory.create(published=True)
+        url = '/project/%s/enrichment?api_key=%s' % (project.short_name, project.owner.api_key)
+        res = self.app_get_json(url)
+        data = json.loads(res.data)
+        assert res.status_code == 200 and 'enrichments' in data, res
+        
+        csrf = data['csrf']
+        enrich_data = {
+            'enrich_data': {
+                'in_field_name': 'A',
+                'task_state': 'e',
+                'type': 'fun',
+                'sub_type': 'picnic',
+                'out_field_name': 'enrich_A'                
+            }
+        }
+        res = self.app.post(url, content_type='application/json',
+                            data=json.dumps(enrich_data),
+                            headers={'X-CSRFToken': csrf})
+        data = json.loads(res.data)
+        assert res.status_code == 200 and \
+            data['flash'] == 'Success! Project data enrichment updated', res
+
+        project = project_repo.get(project.id)
+        updated_enrichments = project.info['enrichments']
+        assert all([True for k in enrich_data if k in updated_enrichments and \
```

Comment:
I think this assertion can never fail. So it's not really asserting anything.
If the enrichment update fails then this will do assert on `all([])` which is `True`.

Reply:
this test is to make sure what is passed for update is recorded correctly in db.

Reply:
Then do you want `assert enrich_data == updated_enrichments`?

Reply:
yes, this is cleaner. will update.